src/editorwidget/widget.ts

```diff
@@ -19,9 +34,21 @@ import {
 } from '../codemirror/widget';
 
 import {
+  Menu, MenuBar, MenuItem
+} from 'phosphor-menus';
+
+import {
   ABCWidgetFactory, IDocumentModel, IWidgetFactory, IDocumentContext
 } from '../docregistry';
 
+import {
```

Comment:
Thanks for all the input, I really appreciate it. I'll make these changes right now. As for the documentation you mentioned for `createMenu`, would that just be a comment above the declaration or do I need to actually add it somewhere?